src/components/NewGalaxyModal.js

```diff
@@ -186,59 +228,119 @@ export const NewGalaxyModal = _.flow(
-        ['PRESTARTING', () => 'Cloud environment is preparing to start.'],
-        ['STARTING', () => `Cloud environment is starting. ${waitMessage}`],
-        ['RUNNING', () => 'Environment consists of an application and cloud compute.'],
+        ['STOPPED', () => `The cloud compute is paused.`],
+        ['PRESTOPPING', () => 'The cloud compute is preparing to pause.'],
+        ['STOPPING', () => `The cloud compute is pausing. ${waitMessage}`],
+        ['PRESTARTING', () => 'The cloud compute is preparing to resume.'],
+        ['STARTING', () => `The cloud compute is resuming. ${waitMessage}`],
+        ['RUNNING', () => nonStatusSpecificMessage],
         ['ERROR', () => `An error has occurred on your cloud environment.`]
       )
   }
 
+  // TODO Refactor this and the duplicate in NewRuntimeModal.js
+  const renderGalaxyCostBreakdown = (kubernetesRuntimeConfig, dataDiskSize) => {
+    const runningComputeCost = getGalaxyComputeCost({ status: 'RUNNING', kubernetesRuntimeConfig })
+    const pausedComputeCost = getGalaxyComputeCost({ status: 'STOPPED', kubernetesRuntimeConfig })
+
+    return div({
+      style: {
+        backgroundColor: colors.accent(0.2),
+        display: 'flex',
+        borderRadius: 5,
+        padding: '0.5rem 1rem',
+        marginTop: '1rem'
+      }
+    }, [
+      _.map(({ cost, label, unitLabel }) => {
+        return div({ key: label, style: { flex: 1, ...styles.label } }, [
+          div({ style: { fontSize: 10 } }, [label]),
+          div({ style: { color: colors.accent(), marginTop: '0.25rem' } }, [
+            span({ style: { fontSize: 20 } }, [cost]),
+            span([' ', unitLabel])
+          ])
+        ])
+      }, [
+        { label: 'Running cloud compute cost', cost: Utils.formatUSD(runningComputeCost), unitLabel: 'per hr' },
+        { label: 'Paused cloud compute cost', cost: Utils.formatUSD(pausedComputeCost), unitLabel: 'per hr' },
+        { label: 'Persistent disk cost', cost: Utils.formatUSD(getGalaxyDiskCost(dataDiskSize)), unitLabel: 'per hr' }
+      ])
+    ])
+  }
+
+  const renderCloudComputeProfileSection = () => {
+    const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '2rem', alignItems: 'center' }
+    return div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
+      div({ style: styles.headerText }, ['Cloud compute profile']),
+      div({ style: { ...gridStyle, marginTop: '0.75rem' } }, [
+        h(MachineSelector, { value: kubernetesRuntimeConfig, onChange: v => setKubernetesRuntimeConfig(v) })
+      ])
+    ])
+  }
+
+  const renderPersistentDiskSection = () => {
+    const gridStyle = { display: 'grid', gridTemplateColumns: '0.75fr 4.5rem 1fr 5.5rem 1fr 5.5rem', gridGap: '1rem', alignItems: 'center' }
+    return div({ style: { ...styles.whiteBoxContainer, marginTop: '1rem' } }, [
+      div({ style: styles.headerText }, ['Persistent Disk']),
```

Comment:
Perhaps this should be `Persistent disk` to match the capitalization scheme of the rest of the page.

![image](https://user-images.githubusercontent.com/5368863/113600725-6bf72680-960e-11eb-91f2-51f0a5da846f.png)

(I don't know why capitalization sticks out to me so much.. sorry if this is being too nit-picky 😁 )

Reply:
Oh yea totally - for some reason I saw `Persistent Disk` at the same level as `Cloud Environment` 🤦🏼 

It's refreshing to see I'm not the only one with stylistic nits 😊